src/shadowbox/server/manager_service.ts

```diff
@@ -94,13 +98,21 @@ export function bindService(
   apiServer.get(`${apiPrefix}/metrics/enabled`, service.getShareMetrics.bind(service));
   apiServer.put(`${apiPrefix}/metrics/enabled`, service.setShareMetrics.bind(service));
 
-  // Experimental APIs
+  // Redirect former experimental APIs
   apiServer.put(
       `${apiPrefix}/experimental/access-key-data-limit`,
-      service.setAccessKeyDataLimit.bind(service));
+      redirect(`${apiPrefix}/server/access-key-data-limit`));
   apiServer.del(
       `${apiPrefix}/experimental/access-key-data-limit`,
-      service.removeAccessKeyDataLimit.bind(service));
+      redirect(`${apiPrefix}/server/access-key-data-limit`));
+}
+
+// Returns a request handler that redirects a bound request path to `url` with HTTP status `code`.
+function redirect(url: string, code = 301): restify.RequestHandlerType {
```

Comment:
Please use 308 instead to preserve the HTTP method.

Reply:
Done.